.golangci.yml

```diff
@@ -1,16 +1,12 @@
-run:
-  timeout: 10m
+version: "2"
```

Comment:
This file is pretty much entirely formatting changes. 

I did disable 1-2 new linters.